tests/conftest.py

```diff
@@ -158,6 +158,28 @@ def multidict_getversion_callable(multidict_module: ModuleType) -> Callable:
     return multidict_module.getversion
 
 
+def write(cls: Any, proto: int) -> bytes:
```

Comment:
This helper should have a useful name and a docstring. Or move this into the fixture.

Reply:
Refactored away!